docs/admin/licensing.md

```diff
@@ -43,9 +43,9 @@ The following features are available in FlowForge Premium:
 [^Instance-Limit]: The number of Node-RED instances is determined by the license purchased.
 [^Device-Limit]: The number of devices is determined by the license purchased.
 [^License-Overage]: The Premium License permits the platform to continue to operate even if the license limits are exceeded. This is to allow for smooth and continuous operation. Overage will be logged and reported to the license holder and FlowForge Inc. The license holder will need to level up their license at the next available opportunity.
-[^License-Expiry]: The Premium License is valid for 12 months from the date of purchase. The FlowForge Platform will not halt operation when the license expires, but will continue to operate to allow for smooth and continuous operation. Expiry will be logged and reported to the license holder and FlowForge Inc. The license holder will need to renew the license to continue using the platform or revert to the Open-Source version.
-[^intercommunication]: FlowForge Project Nodes provide seamless passing of data and messages between your Node-RED instances.
+[^License-Expiry]: The Premium License is valid for 12 months from the date of purchase. The FlowFuse Platform will not halt operation when the license expires, but will continue to operate to allow for smooth and continuous operation. Expiry will be logged and reported to the license holder and FlowForge Inc. The license holder will need to renew the license to continue using the platform or revert to the Open-Source version.
```

Comment:
Just double checking - the use of FlowForge Inc in these 2 sentences are still valid and correct?

Reply:
FlowForge Inc. is still the legal entity, so I would keep it as it is for now.